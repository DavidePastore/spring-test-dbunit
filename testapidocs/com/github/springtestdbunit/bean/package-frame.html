<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!--NewPage-->
<HTML>
<HEAD>
<!-- Generated by javadoc (build 1.6.0_26) on Sat Apr 21 14:47:47 PDT 2012 -->
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>
com.github.springtestdbunit.bean (Spring Test DBUnit 1.0.1-SNAPSHOT Test API)
</TITLE>

<META NAME="date" CONTENT="2012-04-21">

<LINK REL ="stylesheet" TYPE="text/css" HREF="../../../../stylesheet.css" TITLE="Style">


</HEAD>

<BODY BGCOLOR="white">
<FONT size="+1" CLASS="FrameTitleFont">
<A HREF="../../../../com/github/springtestdbunit/bean/package-summary.html" target="classFrame">com.github.springtestdbunit.bean</A></FONT>
<TABLE BORDER="0" WIDTH="100%" SUMMARY="">
<TR>
<TD NOWRAP><FONT size="+1" CLASS="FrameHeadingFont">
Classes</FONT>&nbsp;
<FONT CLASS="FrameItemFont">
<BR>
<A HREF="DatabaseConfigBeanTests.html" title="class in com.github.springtestdbunit.bean" target="classFrame">DatabaseConfigBeanTests</A>
<BR>
<A HREF="DatabaseDataSourceConnectionFactoryBeanTests.html" title="class in com.github.springtestdbunit.bean" target="classFrame">DatabaseDataSourceConnectionFactoryBeanTests</A></FONT></TD>
</TR>
</TABLE>


</BODY>
</HTML>
