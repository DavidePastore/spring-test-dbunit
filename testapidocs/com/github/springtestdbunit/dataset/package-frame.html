<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_06) on Wed Jan 09 20:24:56 PST 2013 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>com.github.springtestdbunit.dataset (Spring Test DBUnit 1.0.1 Test API)</title>
<meta name="date" content="2013-01-09">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/springtestdbunit/dataset/package-summary.html" target="classFrame">com.github.springtestdbunit.dataset</a></h1>
<div class="indexContainer">
<h2 title="Classes">Classes</h2>
<ul title="Classes">
<li><a href="FlatXmlDataSetLoaderTests.html" title="class in com.github.springtestdbunit.dataset" target="classFrame">FlatXmlDataSetLoaderTests</a></li>
</ul>
</div>
</body>
</html>
