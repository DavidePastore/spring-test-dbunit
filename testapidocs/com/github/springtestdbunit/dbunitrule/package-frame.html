<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_06) on Wed Jan 09 20:44:32 PST 2013 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>com.github.springtestdbunit.dbunitrule (Spring Test DBUnit 1.0.2-SNAPSHOT Test API)</title>
<meta name="date" content="2013-01-09">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/springtestdbunit/dbunitrule/package-summary.html" target="classFrame">com.github.springtestdbunit.dbunitrule</a></h1>
</body>
</html>
