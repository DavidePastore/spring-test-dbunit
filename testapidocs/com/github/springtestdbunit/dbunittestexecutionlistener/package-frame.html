<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_06) on Wed Jan 09 20:24:56 PST 2013 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>com.github.springtestdbunit.dbunittestexecutionlistener (Spring Test DBUnit 1.0.1 Test API)</title>
<meta name="date" content="2013-01-09">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/springtestdbunit/dbunittestexecutionlistener/package-summary.html" target="classFrame">com.github.springtestdbunit.dbunittestexecutionlistener</a></h1>
</body>
</html>
