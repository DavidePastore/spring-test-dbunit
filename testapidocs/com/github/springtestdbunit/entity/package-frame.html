<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_06) on Wed Jan 09 20:24:56 PST 2013 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>com.github.springtestdbunit.entity (Spring Test DBUnit 1.0.1 Test API)</title>
<meta name="date" content="2013-01-09">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/springtestdbunit/entity/package-summary.html" target="classFrame">com.github.springtestdbunit.entity</a></h1>
<div class="indexContainer">
<h2 title="Classes">Classes</h2>
<ul title="Classes">
<li><a href="EntityAssert.html" title="class in com.github.springtestdbunit.entity" target="classFrame">EntityAssert</a></li>
<li><a href="HibernateImportTest.html" title="class in com.github.springtestdbunit.entity" target="classFrame">HibernateImportTest</a></li>
<li><a href="SampleEntity.html" title="class in com.github.springtestdbunit.entity" target="classFrame">SampleEntity</a></li>
</ul>
</div>
</body>
</html>
