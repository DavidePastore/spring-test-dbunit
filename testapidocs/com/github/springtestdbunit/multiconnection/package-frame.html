<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (1.8.0_20) on Thu Jan 08 15:11:38 PST 2015 -->
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>com.github.springtestdbunit.multiconnection (Spring Test DBUnit 1.2.0-SNAPSHOT Test API)</title>
<meta name="date" content="2015-01-08">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
<script type="text/javascript" src="../../../../script.js"></script>
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/springtestdbunit/multiconnection/package-summary.html" target="classFrame">com.github.springtestdbunit.multiconnection</a></h1>
<div class="indexContainer">
<h2 title="Classes">Classes</h2>
<ul title="Classes">
<li><a href="MultiConnectionTests.html" title="class in com.github.springtestdbunit.multiconnection" target="classFrame">MultiConnectionTests</a></li>
</ul>
</div>
</body>
</html>
