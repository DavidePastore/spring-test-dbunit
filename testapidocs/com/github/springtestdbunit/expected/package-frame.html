<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_51) on Wed Jan 29 11:32:14 PST 2014 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>com.github.springtestdbunit.expected (Spring Test DBUnit 1.1.1-SNAPSHOT Test API)</title>
<meta name="date" content="2014-01-29">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/springtestdbunit/expected/package-summary.html" target="classFrame">com.github.springtestdbunit.expected</a></h1>
<div class="indexContainer">
<h2 title="Classes">Classes</h2>
<ul title="Classes">
<li><a href="DoesNotSwallowExpectedFailureTest.html" title="class in com.github.springtestdbunit.expected" target="classFrame">DoesNotSwallowExpectedFailureTest</a></li>
<li><a href="ExpectedFailureOnClassTest.html" title="class in com.github.springtestdbunit.expected" target="classFrame">ExpectedFailureOnClassTest</a></li>
<li><a href="ExpectedFailureOnMethodTest.html" title="class in com.github.springtestdbunit.expected" target="classFrame">ExpectedFailureOnMethodTest</a></li>
<li><a href="ExpectedNonStrictOnClassTest.html" title="class in com.github.springtestdbunit.expected" target="classFrame">ExpectedNonStrictOnClassTest</a></li>
<li><a href="ExpectedNonStrictOnMethodTest.html" title="class in com.github.springtestdbunit.expected" target="classFrame">ExpectedNonStrictOnMethodTest</a></li>
<li><a href="ExpectedQueryFailureOnMethodTest.html" title="class in com.github.springtestdbunit.expected" target="classFrame">ExpectedQueryFailureOnMethodTest</a></li>
<li><a href="ExpectedQueryNonStrictFailureOnMethodTest.html" title="class in com.github.springtestdbunit.expected" target="classFrame">ExpectedQueryNonStrictFailureOnMethodTest</a></li>
<li><a href="ExpectedQueryNonStrictOnMethodTest.html" title="class in com.github.springtestdbunit.expected" target="classFrame">ExpectedQueryNonStrictOnMethodTest</a></li>
<li><a href="ExpectedQueryOnMethodTest.html" title="class in com.github.springtestdbunit.expected" target="classFrame">ExpectedQueryOnMethodTest</a></li>
<li><a href="ExpectedTableNonStrictOnClassTest.html" title="class in com.github.springtestdbunit.expected" target="classFrame">ExpectedTableNonStrictOnClassTest</a></li>
</ul>
</div>
</body>
</html>
